miniver/app.py

```diff
@@ -90,10 +90,7 @@ def _line_in_file(to_find, filename):
     assert "\n" not in to_find
     try:
         with open(filename) as f:
-            for line in f:
-                if to_find in line:
-                    return True
-            return False
+            any(to_find in line for line in f)
```

Comment:
There needs to be a return statement here. Also note: with this implementation the entire file is read, where as the current master can return early.

Reply:
Indeed! Fixed.